source/install/config-ssl-http2-nginx.rst

```diff
@@ -0,0 +1,95 @@
+.. _config-ssl-http2-nginx:
+
+Configuring NGINX with SSL and HTTP/2
+=====================================
+
+Using SSL gives greater security by ensuring that communications between Mattermost clients and the Mattermost server are encrypted. It also allows you to configure NGINX to use the HTTP/2 protocol.
+
+Although you can configure HTTP/2 without SSL, both Firefox and Chrome browsers support HTTP/2 on secure connections only.
+
+You can use any certificate that you want, but these instructions show you how to download and install certificates from *Let's Encrypt*.
+
+**To configure SSL and HTTP/2:**
+
+1. Log into the server that hosts NGINX and open a terminal window.
+2. Install git.
+
+  - If you are using Ubuntu or Debian:  ``sudo apt-get install git``
```

Comment:
I changed this to be consistent with the rest of the docs. See commit https://github.com/JeffSchering/docs/commit/787e1d004100f8fb2fdb4a0c1d120a704ade5dfe